app/classifier/tasks/shortcut.jsx

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import { Markdown } from 'markdownz';
+
+class Summary extends React.Component {
+  render() {
+    let answer;
+
+    if (this.props.annotation.value) {
+      answer = (
+        <div className="answer">
+          <i className="fa fa-check-circle-o fa-fw" />
+          <Markdown tag="span" inline={true}>{this.props.task.answers[this.props.annotation.value].label}</Markdown>
+        </div>
+      );
+    } else {
+      answer = <div className="answer">No answer</div>;
+    }
+
+    return (
+      <div>
+        <div className="question">
+          {this.props.task.question}
+        </div>
+        <div className="answers">
+          {answer}
+        </div>
+      </div>
+    );
+  }
+}
+
+Summary.propTypes = {
+  annotation: React.PropTypes.shape(
+    { value: React.PropTypes.number }
+  ).isRequired,
+  task: React.PropTypes.shape(
+    {
+      answers: React.PropTypes.array,
+      question: React.PropTypes.string
+    }
+  )
+};
+
+Summary.defaultProps = {
+  annotation: null,
```

Comment:
A default prop of null will throw console warnings if the prop is also required. Either prop can't be required or this needs to be set to a default value.